src/scene/morph-instance.js

```diff
@@ -183,10 +186,11 @@ class MorphInstance {
     /**
      * Sets weight of the specified morph target.
      *
-     * @param {number} index - An index of morph target.
+     * @param {number|string} key - An identifier for the morph target. Either the weight index or the weight name
      * @param {number} weight - Weight.
      */
-    setWeight(index, weight) {
+    setWeight(key, weight) {
```

Comment:
Just throwing it out there, but does it make sense to have two functions? I'd be interested to hear @mvaligursky's preference since he has experience with this API.

Reply:
I thought about it yesterday, and I think I'm ok with this - the API is not used directly all that much, usually I'd expect to be driven by the animation, and an extra dictionary look up is not a big deal.